crates/ark/src/reticulate.rs

```diff
@@ -11,6 +11,7 @@ use harp::utils::r_is_null;
 use harp::RObject;
 use libr::R_NilValue;
 use libr::SEXP;
+use once_cell::sync::Lazy;
```

Comment:
This should now be `LazyCell` from the std lib

Reply:
I had to use `LazyLock` instead (https://github.com/posit-dev/ark/pull/759/commits/a616063a32b215fac5d079076892f344c0f96db2) because `LazyCell` is not safe to be shared as a global.